libs/community/langchain_community/document_loaders/dedoc.py

```diff
@@ -0,0 +1,416 @@
+                    raw text nodes)
+                "line": split document into lines
+            with_tables: add tables to the result - each table is returned as a single
+                langchain Document object
+
+            dedoc_kwargs: parameters used for document parsing via `dedoc`
+                (https://dedoc.readthedocs.io/en/latest/parameters/parameters.html).
+                with_attachments: enable attached files extraction
+                recursion_deep_attachments: recursion level for attached files
+                    extraction, works only when with_attachments==True
+                pdf_with_text_layer: type of handler for parsing PDF documents,
+                    available options
+                    ["true", "false", "tabby", "auto", "auto_tabby" (default)]
+                language: language of the document for PDF without a textual layer and
+                    images, available options ["eng", "rus", "rus+eng" (default)],
+                    the list of languages can be extended, please see
+                    https://dedoc.readthedocs.io/en/latest/tutorials/add_new_language.html
+                pages: page slice to define the reading range for parsing PDF documents
+                is_one_column_document: detect number of columns for PDF without
+                    a textual layer and images, available options
+                    ["true", "false", "auto" (default)]
+                document_orientation: fix document orientation (90, 180, 270 degrees)
+                    for PDF without a textual layer and images, available options
+                    ["auto" (default), "no_change"]
+                need_header_footer_analysis: remove headers and footers from the output
+                    result for parsing PDF and images
+                need_binarization: clean pages background (binarize) for PDF without a
+                    textual layer and images
+                need_pdf_table_analysis: parse tables for PDF without a textual layer
+                    and images
+                delimiter: column separator for CSV, TSV files
+                encoding: encoding of TXT, CSV, TSV
+        """
+        super().__init__(
+            file_path=file_path, split=split, with_tables=with_tables, **dedoc_kwargs
+        )
+        self.url = url
+        self.parsing_parameters["return_format"] = "json"
+
+    def lazy_load(self) -> Iterator[Document]:
+        """Lazily load documents."""
+        doc_tree = self._send_file(
+            url=self.url, file_path=self.file_path, parameters=self.parsing_parameters
+        )
+        yield from self._split_document(document_tree=doc_tree, split=self.split)
+
+    def _make_config(self) -> dict:
+        return {}
+
+    def _send_file(
+        self, url: str, file_path: str, parameters: dict
+    ) -> Dict[str, Union[list, dict, str]]:
+        """Send POST-request to `dedoc` API and return the results"""
+        import json
```

Comment:
nit: These are standard library imports you can push them to the global scope

Reply:
Thank you, we'll fix it!